cmd/subctl/join.go

```diff
@@ -67,10 +67,10 @@ var joinCmd = &cobra.Command{
 		clientProducer, err := client.NewProducerFromRestConfig(clientConfig.Config)
 		exit.OnError(status.Error(err, "Error creating the client producer"))
 
-		client, err := controllerClient.New(clientConfig.Config, controllerClient.Options{})
+		client, err := controllerclient.New(clientConfig.Config, controllerclient.Options{})
```

Comment:
Can’t we use the general client from `clientProducer` here?

Reply:
The `clientProducer` created on L67 is still the **submariner-operator** version. I'd have to migrate all the `join` code to use the new `clientProducer` but this PR is intended to adjust to the **submariner-operator** changes so wanted to keep the changes here to a minimum.

Reply:
Ah right, I see!